File: prog/python/substring.py
s1=input("Enter a String:")
s2=input("Enter Substring:")
count=0
for i in range(0,len(s1)-len(s2)+1):
    j=0
    b=0
    if(s1[i]==s2[j]):
        k=i
        while(j<len(s2)):
            if(s1[k]!=s2[j]):
                b=1
                break
            j=j+1
            k=k+1
        if(b==0):
            print("The Sub string is at {}".format(i))
            break
else:
    print("Sub String Not found")

            
            
				

File: prog/c/substring.c
#include<stdio.h>
#include<string.h>
int main()
{
	char str[50],s[30];
	int i,j,k,b;
	fgets(str,50,stdin);
	fgets(s,50,stdin);
	for(i=0;i<=strlen(str)-strlen(s);i++)
	{
		j=0;
		b=0;

		if(str[i]==s[j])
		{
			for(k=i;j<=strlen(s)-1;k++,j++)
			{
				if(str[k]!=s[j])
				{	
					b=1;	
					break;
				}
			}
			if(b==0)
			{
				printf("Sub string is at %d",i);
				return 0;
			}
		}
	}
	printf("sub String not found");
	return 0;
}

	
